content/en/about/bundle.md

```diff
@@ -0,0 +1,313 @@
+# Sigstore Bundle
+
+October 31, 2024
+
+Version 0.3.2
+
+This document describes the data structure for storing Sigstore signatures generated by tooling
+working in the context of the Sigstore Public Instance. It includes `json` examples of
+serialized bundles of the current bundle format version. It may exclude descriptions of parameters
+that continue to exist for compatibility reasons or for private use cases. For a full description of
+the format, the formal schema and information about language library support see
+[sigstore/protobuf-specs](https://github.com/sigstore/protobuf-specs).
+
+## Bundle
+
+A Sigstore bundle is everything required to verify a signature on an artifact. This
+is satisfied by the **Verification Material** and signature **Content**.
+
+### Verification Material
+
+This is key material used to verify signatures along with supporting metadata like transparency log entries and timestamps. When using short lived Fulcio certificates where verification may occur after the certificate has expired, bundles must include at least one transparency log's signed entry timestamp or an [RFC3161](https://www.ietf.org/rfc/rfc3161.txt) timestamp to provide proof that signing occured during the ceritificates validity window.
+
+#### Key Material
+
+##### X.509 certificate
+
+A single X.509 leaf certificate conveying the signing key and containing [extensions](https://github.com/sigstore/fulcio/blob/main/docs/oid-info.md)
+for identities consumed at verification time. This is the recommended `"verificationMaterial"` type
+for use with the public Sigstore infrastructure.
+
+```json
+"verificationMaterial": {
+  "certificate": {
+    "rawBytes": "<Base64(DER bytes)>"
+  }
+}
+```
+
+##### Public Key Identifier
+
+A hint to identify an out of band delivered key to verify a signature. Like traditional PKI key distribution
+the format of the hint must be agreed upon out of band by the signer and the verifiers. The key itself is not embedded in the Sigstore bundle.
+
+```json
+"verificationMaterial": {
+  "publicKeyIdentifier": {
+    "hint": "<HINT>"
+  }
+}
+```
+
+#### Transparency Log Entries
+
+One or more transparency logs entries to provide proof of inclusion in a public log and optionally a timestamp to
```

Comment:
Minor: a transparency log entry isn't required for a bundle, should we say "Zero or more..." like we do below for Timestamps?

We already say above:
> bundles must include at least one transparency log's signed entry timestamp or an RFC3161 timestamp

Reply:
Oh I always thought a log entry was required but you don't have to trust the logs clock (so you can ignore the SET and use the rfc3161 timestamp). Maybe that's only true for the public usecase?

Reply:
Log entries are not required - https://github.com/sigstore/protobuf-specs/blob/main/protos/sigstore_bundle.proto#L105

I think we should word it that it is expected that bundles contain log entries for public consumption.

Reply:
updated this section a bit